types/cache-interceptor.d.ts

```diff
@@ -6,7 +6,7 @@ declare namespace CacheHandler {
   export type CacheMethods = 'GET' | 'HEAD' | 'OPTIONS' | 'TRACE'
 
   export interface CacheOptions {
-    store?: CacheStore
+    store: CacheStore
```

Comment:
This is incorrect, we default to memory.

Reply:
Not according the cache handler. Then we have a different type issue

Reply:
https://github.com/nodejs/undici/blob/a78147362135d7e5ea1617cef093fe5512a27ad9/lib/interceptor/cache.js#L90-L93 doesn't look that way.